iOS_4x_release.md

```diff
@@ -0,0 +1,48 @@
+# Aligning major version releases for the Adobe iOS SDK
+
+### Why are we getting another major update already?
+
+Apple has started doing an annual requirement to increase the minimum version of Xcode used when [submitting to the App Store](https://developer.apple.com/ios/submit/). Each Xcode version only supports back to a specific minimum version of iOS. Loosely speaking, the Xcode version and _minimum_ iOS version are correlated. 
+
+In order to align with Apple's changing requirements, we will also be doing an annual major release of the Adobe Experience Platform Mobile SDKs for iOS. In this yearly major update, you can expect the following changes:
+ - Increasing the minimum supported version of iOS.
+ - Increasing the minimum supported version of Xcode.
+ - Possible increase to minimum supported version of Swift.
+
+### Why are all the extensions updating to major version 4?
+
+The main goal for this change is to simplify Adobe Mobile SDK distribution. Aligning the extensions on the same major version allows us to create predictability. For example:
+ - All Adobe-owned mobile SDK extensions that share a major version are compatible with one another.
+ - Each major version of an Adobe-owned mobile SDK extension has a designated minimum supported OS version.
+
+### Why did some of the extensions skip major versions?
+
+You may be wondering "what happened to versions 2 and 3 of AEPEdge?" 
+
+In order to achieve our goal of unifying major versions, AEPEdge (and a handful of other extensions) had to skip one or more major versions. Don't worry, you didn't miss them.
+
+### What will happen with new extensions?
+
+A new extension - that is, an extension getting its first public release - will be released with the same major version as AEPCore. As part of this major update we are introducing a new extension, AEPEdgeMedia. As you would expect, the first version of this extension is 4.0.0.
+
+### Do I have to reimplement the SDK?
+
+Unlike our previous major version updates, this is not a major rewrite nor does it include major feature updates or SDK code changes. You can expect all of your API calls in the previous major version of the SDK to continue working with the new version (with the exception of previously deprecated APIs - more on that later).
+
+While we don't have plans for another major rewrite any time soon, if a need for one comes up, we will be sure to let you know with plenty of lead time.
+
+Outside of major version updates, all of our extensions will continue to follow semantic versioning ([semver](https://semver.org/)) for MINOR and PATCH releases on an as-needed basis. We recommend using the latest version of each extension, which will contain the most current optimizations and bug fixes.
+
+### How does this impact the SDK going forward?
+
+For starters, having more frequent updates allows us to better take advantage of new frameworks and APIs introduced by Apple. This will help us achieve a more modern SDK.
```

Comment:
"This will help us remain a modern SDK that remains consistent and in lock-step with changes in iOS"